clang/tools/IndexStore/CMakeLists.txt

```diff
@@ -28,11 +23,10 @@ add_clang_library(IndexStore SHARED
   ${SOURCES}
 
   LINK_LIBS
-  ${LIBS}
+  clangIndex
+  clangIndexDataStore
 
   LINK_COMPONENTS
-  ${LLVM_TARGETS_TO_BUILD}
-  Core
```

Comment:
In `clang/lib/Index/` we have

```
set(LLVM_LINK_COMPONENTS
  BitReader
  BitstreamReader
  Core
  Support
  )
```

So I'm not sure we'll actually drop Core in practice.  It seems like the real change here is LLVM_TARGETS_TO_BUILD?

Reply:
I just dropped Core here as well since `clang/tools/IndexStore` doesn't directly depend on it. If it comes through clangIndex that's fine IMO.

But yes, the real change here is `LLVM_TARGETS_TO_BUILD`.